plugins/flutter_aepedgeconsent/pubspec.lock

```diff
@@ -57,9 +57,10 @@ packages:
   flutter_aepcore:
     dependency: "direct main"
     description:
-      path: "../flutter_aepcore"
-      relative: true
-    source: path
+      name: flutter_aepcore
+      sha256: f86c1ab15bd0942922584e4860f583967ecfdaa7b1e9be0085f9d19a2e3ec6d9
```

Comment:
some of these other plugins got updated dependencies for core from url instead of relative, is this intended for this PR?

Reply:
I'm confused by this as well, the pubspec override isn't working?

Reply:
This is intended, we took out the dependency_overrides for plugins when merge from staging to main last time.
https://github.com/adobe/aepsdk_flutter/pull/58/files
It didn't run get pub after changes, so the pubspec.lock didn't reflect it.  
But it reflected the change when we run it now.